README.md

```diff
@@ -19,15 +19,15 @@ To view the website offline one your own machine, you will need to clone it loca
 
 ## Formatting and style
 
-We use [GitHub markdown](https://guides.github.com/features/mastering-markdown/) to format the basic text. Have a look at the markdown in existing pages for examples. 
+We use [GitHub markdown](https://guides.github.com/features/mastering-markdown/) to format the basic text. Have a look at the markdown in existing pages for examples.
 
-Note that we use British English spelling and grammar conventions. 
+Note that we use British English spelling and grammar conventions.
```

Comment:
(Despite the language dropdown in Jamulus claiming to be American English...)

Reply:
I think I recall corrados wanting to use en_US spelling in the app. Don't know how consistent it is there though.